source/_integrations/ecovacs.markdown

```diff
@@ -23,90 +24,15 @@ There is currently support for the following device types within Home Assistant:
 
 {% include integrations/config_flow.md %}
 
-Note: For some countries, you will need to set `continent` to `ww` (meaning worldwide.) There is unfortunately no way to know the correct settings other than guessing and checking. See the [py-sucks library protocol documentation](https://github.com/mib1185/py-sucks/blob/master/protocol.md) for more information about what has been figured out about the Ecovacs servers.
+Note: For some countries, you will need to set `continent` to `ww` (meaning worldwide). There is unfortunately no way to know the correct settings other than guessing and checking.
```

Comment:
Shouldn't we remove the mention of continent?

Reply:
Yes will do it in a follow up

Reply:
https://github.com/home-assistant/home-assistant.io/pull/30959